api/v1/runtime/common.proto

```diff
@@ -171,6 +171,7 @@ message Netflow {
   string node_name = 13;
   bytes addr = 9;
   uint32 port = 10;
+  uint32 pid = 14;
```

Comment:
We need to have pid in order to understand unique flows. Currently we have multiple flows with the same sources and destinations for the same time period. Local testing showed that local pid actually changes (make sense).